### PR TITLE
MS-718 Who do you work for? Page

### PR DESCRIPTION
**Pull request by jjrobothamho**
Who do you work for? Page created + acceptance test updated.

### CODE REVIEW

File: apps/nrm/index.js [hunk 1]
```diff
@@ -20,7 +20,27 @@ module.exports = {
       next: '/pv-under-age'
     },
     '/pv-under-age': {
-      next: '/pv-under-age-at-time-of-exploitation'
+      fields: ['pv-under-age'],
```
Comment (sulthan-ahmed) [inline apps/nrm/index.js]: is this from your other branch?

File: apps/verify/fields/index.js [hunk 2]
```diff
@@ -1,6 +1,17 @@
 'use strict';
 
+const organisations = require('ms-organisations');
+
```
Comment (sulthan-ahmed) [inline apps/verify/fields/index.js]: 👍

File: apps/nrm/fields/index.js [hunk 3]
```diff
@@ -11,9 +11,29 @@ module.exports = {
       'england',
       'wales',
       'scotland',
-      'northernireland'
+      'northern-ireland'
     ]
   },
+  'pv-under-age': {
```
Comment (sulthan-ahmed) [inline apps/nrm/fields/index.js]: Is this from your previous branch?

File: acceptance-test/user-pathways/happy-path/happy-path.test.js [hunk 4]
```diff
@@ -50,7 +51,14 @@ describe('Critical user path(s)', () => {
 
     it('Happy path - Adult', async() => {
         try {
+            // start
             await clickContinueButton(1);
+
+            // who-do-you-work-for
+            await page.waitForSelector(ORGANISATION_INPUT);
+            await page.$eval(ORGANISATION_INPUT, (element) => {
+                element.value = 'Barnardos';
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/happy-path/happy-path.test.js]: This suggests ms-organisations have the format of first letter uppercase. Is that consistent throughout the app? E.g. you have england as first letter lowercase

I’d probably stick to lowercase

File: apps/verify/views/partials/find-organisation.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<details class="govuk-details">
+    <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+        I can’t find my organisation
+        </span>
+    </summary>
+    <p>You must work for a designated first responder organisation to use this service.</p>
+    <p>If you're not a first responder, you can still report modern slavery by calling the modern slavery helpline on 0800 0121 700 or <a href="https://www.modernslaveryhelpline.org/report">reporting it online</a>.</p>
+</details>
+<br/>
```
Comment (sulthan-ahmed) [inline apps/verify/views/partials/find-organisation.html]: New line?

File: apps/verify/views/who-do-you-work-for.html [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#renderField}}user-organisation{{/renderField}}
+    {{> partials-find-organisation}}
+    {{#renderField}}user-email{{/renderField}}
+    {{#input-submit}}continue{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```
Comment (sulthan-ahmed) [inline apps/verify/views/who-do-you-work-for.html]: New line?

File: apps/nrm/index.js [hunk 7]
```diff
@@ -20,7 +20,27 @@ module.exports = {
...
+        }
+       },
+       {
+        target: '/local-authority-contacted-about-child',
+        condition: {
+          field: 'pv-under-age',
+          value: 'not-sure'
+        }
+       }
+      ],
```
Comment (sulthan-ahmed) [inline apps/nrm/index.js]: Should this forking be part of your other PR?

File: apps/verify/fields/index.js [hunk 8]
```diff
@@ -1,6 +1,17 @@
...
 module.exports = {
+  'user-organisation': {
+    mixin: 'select',
+    validate: 'required',
+    className: ['typeahead', 'js-hidden'],
+    options: [{
+      value: '',
+      label: 'fields.user-organisation.options.null'
+    }].concat(organisations)
+  },
```
Comment (sulthan-ahmed) [inline apps/verify/fields/index.js]: You worked it out, nice

File: apps/nrm/index.js (hunk 1 above)
Comment (jjrobothamho) [reply apps/nrm/index.js]: Sorry PR'd this one too quickly. Once the other PR is closed these commits will disappear.

**Comment by sulthan-ahmed** [comment]
Would you be able to explain why you closed this PR? It just helps if someone is wondering why

**Pull request closed without merging**